### DISCUSSION TITLE
New Way To Clone A Repository

### DISCUSSION THREAD

**Discussion by vvhybe** [community/community#53547] [Repositories]
### Select Topic Area

Product Feedback

### Body

@Github instead of using the traditional `HTTPS` cloning form 😒:
```bash
git clone `https://github.com/`username/repo.git
````
Use my alternative idea 💡, it consists of making it similar to 👀 Git CLI and SSH by replacing the repetitive 🔁 github domain `https://github.com/` so it will look like this 🦉:
```bash
git clone `username@repo.git`
```
or even:
```bash
git clone `username@repo`
```
> I hope everyone shares the same idea as mine. 🧭💡
I mean I hope everyone shares the same laziness as mine 😂💤

**Comment by airtower-luna** [comment]
That is not going to work for Git, because it would break with repositories hosted anywhere other than GitHub. And Git already interprets `user@host` as the start of an SSH URL. :wink: 

Something similar *can* work with a dedicated tool for GitHub. For example, with the GitHub CLI [`gh repo clone`](https://cli.github.com/manual/gh_repo_clone) does exactly what you want.

**Reply by vvhybe** [reply]
Yes, now when you think about it will somehow break with hosted repositories as you say, my idea wasn't to refactor the way how you clone over the HTTPS method because there is already GithubCLI `gh repo clone` for it, but the way I was talking about it's just an alias for the repo link
like you put the HTTPS repo link inside a variable or something like that.
like when you write 
```bash
git clone user@repo
```
it will be refactored as:
```bash
git clone https://github.com/`user`/`repo`.git
```
As I said it is more like an alias or a shortcut for it.
